src/app/containers/MediaPage/index.stories.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import MediaPage from '.';
+import indonesia from '../../../../data/indonesia/bbc_indonesian_radio/liveradio.json';
+import korean from '../../../../data/korean/bbc_korean_radio/liveradio.json';
+import tigrinya from '../../../../data/tigrinya/bbc_tigrinya_radio/liveradio.json';
+import afaanoromoo from '../../../../data/afaanoromoo/bbc_afaanoromoo_radio/liveradio.json';
+import amharic from '../../../../data/amharic/bbc_amharic_radio/liveradio.json';
+
+const liveRadioFixtures = {
+  indonesia,
+  korean,
+  tigrinya,
+  afaanoromoo,
+  amharic,
+};
+
+const matchFixtures = service => ({
+  params: {
+    mediaId: 'liveradio',
+    serviceId: {
+      indonesia: 'bbc_indonesian_radio',
+      korean: 'bbc_korean_radio',
+      tigrinya: 'bbc_tigrinya_radio',
+      afaanoromoo: 'bbc_afaanoromoo_radio',
+      amharic: 'bbc_amharic_radio',
+    }[service],
+  },
+});
+
+storiesOf('Pages|Media Page', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
```

Comment:
Can we use `dirDecorator` here. [docs](https://www.npmjs.com/package/@bbc/psammead-storybook-helpers#dirdecorator)

Reply:
@sadick254 had a look at `dirDecorator` and I don't think we can use it, unless I'm misunderstanding it

We need to pass all 4 arguments to `inputProvider` - it seems `dirDecorator` just passes 2 and lets the other 2 be defaulted

We need the 3rd argument (line 55) to constrain the services to the 5 that we support, and we need the 4th arg (line 56) to prevent it from defaulting to `news` (which we don't support)

Reply:
I overlooked line 55 and 56. This looks good to me.